memo.rb

```diff
@@ -5,16 +5,20 @@
 
 CONN = PG.connect(dbname: 'sinatra_memo_db')
 
-def conn
-  CONN
+def load_memo
+  CONN.exec_params('SELECT * FROM memos WHERE id=$1', load_params.values_at(:id)).first
```

Comment:
```suggestion
def load_memo(id)
  CONN.exec_params('SELECT * FROM memos WHERE id = $1', [id]).first
```

引数で渡す方がメソッドの柔軟性が上がります。
あと `=` （というか演算子全般）の前後はスペースを入れることが多いですね。